bin/run.sh

```diff
@@ -1,37 +1,33 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e # Make script exit when a command fail.
 set -u # Exit on usage of undeclared variable.
 # set -x # Trace what gets executed.
 
-# Command line arguments
 SLUG="$1"
 INPUT_DIR="$2"
 OUTPUT_DIR="$3"
 
-# Setup a working directory
-WORK_DIR=/opt/representer/app
-cp -rp $INPUT_DIR $WORK_DIR
+# Normalize identifiers and remove comments
+ELM_FILES=("$INPUT_DIR"/*.elm)
+if [ ${#ELM_FILES[@]} == 0 ]; then
+    echo "No .elm file found (${ELM_FILES[@]})"
+    exit -1
+elif [ ${#ELM_FILES[@]} -gt 1 ]; then  
+    echo "Multiple .elm files found (${ELM_FILES[@]})"
+    exit -2
```

Comment:
This is absolutely fine. You could consider concatenating the representation of the multiple files, but there is little chance that their representation would ever match another solution using multiple files.

Reply:
Hi Erik, the code doesn't currently support multiple files. Concatenating the multiple normalizations of the code would be ok, but concatenating the mapping.json wouldn't work, as there could be duplicates. Potentially we should do something different than just returning an error though. Would it be better to just return the concatenated files as representation.txt and "@" for mapping.json?

Reply:
Hi, I still have to properly review the code but regarding multiple files, if returning an error is ok, I think we shouldn't care too much. @ceddlyburge , you could also use the previous version of this representer that strip comments, format the code and concatenate the normalized elm files.

Reply:
I think returning an error is fine.